### PR TITLE
Updating samples

### PR DESCRIPTION
**Pull request by fao89**
- remove the symlinks
- update IPAM sample

### CODE REVIEW

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 1]
```diff
@@ -3,15 +3,11 @@ kind: OpenStackDataPlaneNodeSet
 metadata:
   name: openstack-edpm-ipam
 spec:
-  ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
   env:
     - name: ANSIBLE_FORCE_COLOR
       value: "True"
     - name: ANSIBLE_ENABLE_TASK_DEBUGGER
       value: "True"
-  preProvisioned: true
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:12]: What's the reason for dropping this? We should decide on what we want the samples to represent, all fields with default values, or the minimal set of fields. For instance, why drop preProvisioned, but keep env?

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 2]
```diff
@@ -21,60 +17,27 @@ spec:
...
+            subnetName: subnet1
+            defaultRoute: true
+            fixedIP: 192.168.122.100
+          - name: InternalApi
+            subnetName: subnet1
+          - name: Storage
+            subnetName: subnet1
+          - name: Tenant
+            subnetName: subnet1
     managementNetwork: ctlplane
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:38]: this is optional. should we drop it?

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 3]
```diff
@@ -21,60 +17,27 @@ spec:
...
     ansible:
-      ansibleUser: cloud-admin
+      ansibleUser: root
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:40]: Aren't we sticking with cloud-admin?

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 4]
```diff
@@ -21,60 +17,27 @@ spec:
...
       ansiblePort: 22
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:41]: this is the default. should we drop it?

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 5]
```diff
@@ -21,60 +17,27 @@ spec:
...
       ansibleVars:
          service_net_map:
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:43]: not used anywhere. can be dropped.

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 6]
```diff
@@ -87,32 +50,11 @@ spec:
          # Default nic config template for a EDPM compute node
          # These vars are edpm_network_config role vars
          edpm_network_config_hide_sensitive_logs: false
+         edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:53]: this is the default. drop it?

**Review by slagle — CHANGES_REQUESTED** [review]

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml (hunk 3 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:40]: I know we use cloud-admin for baremetal, but I believe install_yamls still using root

Comment (bshephar) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:40]: We should probably decide what we actually want here. With tripleo, we went through the stages of `heat-admin` and `tripleo-admin`. I guess we want to standardise on something similar here? But probably not root right?

Understand the install_yamls situation. Just saying that we probably want to aim for a non-root user before we GA.

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml (hunk 6 above)
Comment (bshephar) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:53]: Now that we have merged the new CRD design, I'll keep pushing forward with my other PR to change how this interface works anyway. 
https://github.com/openstack-k8s-operators/dataplane-operator/pull/394

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml [hunk 7]
```diff
@@ -130,11 +72,8 @@ spec:
          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: 127.0.0.1
          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 12345678
          edpm_ovn_metadata_agent_DEFAULT_bind_host: 127.0.0.1
-         ctlplane_dns_nameservers:
-         - 192.168.111.1
-         dns_search_domains: []
          edpm_ovn_dbs:
-         - 192.168.111.1
+         - 192.168.122.1
```
Comment (bshephar) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:76]: I think I had issues with `192.168.122.0/24` which is why I ended up on `192.168.111.0/24` I notice the rdo checks haven't run for this change so we might not be seeing the issue with addressing. Provided this actually works in CI and doesn't require some other change, I have no other preference to any particular subnet. Just calling it out in-case there's more required for this subnet change.

Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml:76]: I think this is another thing that is different when baremetal or not: https://github.com/openstack-k8s-operators/dataplane-operator/commit/9b19caf25ca568012a20738b2b5a12c9579ad4b4

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml [hunk 8]
```diff
@@ -28,7 +28,6 @@ spec:
          hostName: edpm-compute-0
          ansible:
            ansibleHost: 192.168.122.100
-           ansibleUser: root
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml:31]: There's no default set for ansibleUser. Shouldn't it be cloud-admin?

Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml:31]: I'm adding cloud-admin on install_yamls: https://github.com/openstack-k8s-operators/install_yamls/pull/531

Comment (slagle) [reply config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml:31]: As long as it's set here in the samples as well. thanks.

**Pull request merged**